Upstatement/puppy-lib
# Feature: Add support for JS and YAML data files

## What this Does
- Adds support for Javascript modules exporting an object or functions (sync or async, useful for loading API data) as data files
- Adds support for YAML data files
- Updates test coverage

## Steps to Test
1. Checkout this branch and run `npm link` in the project root
2. Switch to any project based on the Puppy Starter and run `npm link @upstatement/puppy` to use your local copy of this package instead of the one from NPM
3. Add some JS and YAML data files (see the [fixtures directory](https://github.com/Upstatement/puppy/tree/feat/6-expand-data-formats/test/fixture/src/data) for inspiration)
